styles/stylesheet.css

```diff
@@ -0,0 +1,407 @@
+*{
+  box-sizing: border-box;
+}
+
+/* ***Colors used in styling site:***
+Green color used in header and for CTAs (#00cc00)
+White font color (#ffffff)
+Blue font used in navigation and headers (#00394d)
+Light grey used for nav bar and borders (#e6e6e6)
+ */
+
+/* Setting the body text to be sans-serif, will use specific class styling as required for serif text */
+body {
+  font-family: 'Open Sans', 'Lato', sans-serif;
+}
+
+/* Removing the underlines from ALL links */
+a {
+  text-decoration: none;
+}
+
+/* Styling the header for images, fonts, etc. */
+header {
+  height: 30rem;
+  background-image: url("../images/header-image.jpg");
+  background-size: cover;
+  display: flex;
+  justify-content: space-between;
+  margin: 0;
+  padding: 5% 5%;
+}
+
+.header-text {
+  color: #00cc00;
+  font-family: "Times New Roman", Times, serif;
+  font-style: italic;
+}
+
+header a {
+  color: #ffffff;
+}
+
+.cart {
+  width: 2rem;
+}
+
+.checkout {
+  background-color: #00cc00;
+  font-size: .75rem;
+  font-weight: 700;
+  padding: .5rem 2rem;
```

Comment:
It's a little thing, but when using numbers that are less than zero, it's nice to include the 0;

``` 
padding: 0.5rem 2 rem;
```

It just helps it look a little nicer, not a big deal, but nice.